pkg/karmadactl/join/join.go

```diff
@@ -159,12 +159,12 @@ func (j *CommandJoinOption) AddFlags(flags *pflag.FlagSet) {
 	flags.StringVar(&j.ClusterProvider, "cluster-provider", "", "Provider of the joining cluster. The Karmada scheduler can use this information to spread workloads across providers for higher availability.")
 	flags.StringVar(&j.ClusterRegion, "cluster-region", "", "The region of the joining cluster. The Karmada scheduler can use this information to spread workloads across regions for higher availability.")
 	flags.StringSliceVar(&j.ClusterZones, "cluster-zones", nil, "The zones of the joining cluster. The Karmada scheduler can use this information to spread workloads across zones for higher availability.")
-	flags.StringVar(&j.HostAs, "host-as", "",
-		"Host cluster username to impersonate for the operation. User could be a regular user or a service account in a namespace.")
-	flags.StringArrayVar(&j.HostAsGroups, "host-as-group", []string{},
-		"Host cluster group to impersonate for the operation, this flag can be repeated to specify multiple groups.")
-	flags.StringVar(&j.HostAsUID, "host-as-uid", "",
-		"Host cluster UID to impersonate for the operation.")
+	flags.StringVar(&j.KarmadaAs, "karmada-as", "",
+		"Karmada cluster username to impersonate for the operation in karmada control plane. User could be a regular user or a service account in a namespace.")
```

Comment:
`Karada cluster username` It sounds a little bit confusing, do you have any suggestion @RainbowMango

Reply:
"Username to impersonate for the operation in karmada control plane" ?

Reply:
> "Username to impersonate for the operation in karmada control plane" ?

+1

Reply:
updated